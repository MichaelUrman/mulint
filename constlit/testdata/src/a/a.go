// Package a tests simple literal replacements
package a

import "image/png"

var anah = -2

var abs png.CompressionLevel = -2 // want "Replace `-2` with `png.BestSpeed`"

func aCompress(level png.CompressionLevel) {}

func amain() {
	aCompress(-3) // want "Replace `-3` with `png.BestCompression`"
}
