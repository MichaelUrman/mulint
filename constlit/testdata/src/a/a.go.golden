// Package a tests simple literal replacements
package a

import "image/png"

var anah = -2

var abs png.CompressionLevel = png.BestSpeed // want "Replace `-2` with `png.BestSpeed`"

func aCompress(level png.CompressionLevel) {}

func amain() {
	aCompress(png.BestCompression) // want "Replace `-3` with `png.BestCompression`"
}
